<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelTask">
        <property name="text">
         <string>Task List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidgetTask"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="labelTaskName">
          <property name="text">
           <string>Task Name: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditTaskName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonSelectSysMat">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButtonSelectSinogram">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelSysMat">
          <property name="text">
           <string>System Matrix: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditSysMat">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditSinogram">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelSinogram">
          <property name="text">
           <string>Sinogram:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Output Dir: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditOutputDir">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonSelectOutputDir">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditNumIters">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>500</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="labelNumDualIters">
          <property name="text">
           <string>Number of Dual Iterations: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="labelIteratpr">
          <property name="text">
           <string>Iterator:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBoxRestore">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Restore with AI</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="labelLambda">
          <property name="text">
           <string>Lambda:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelGamma">
          <property name="text">
           <string>Gamma:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelNumIters">
          <property name="text">
           <string>Number of Iterations: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="lineEditLambda">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0.12</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditGamma">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>0.01</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="comboBoxIterator">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>EM-Tikhonov</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MLEM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-2DWavelet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-Cont</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-Cont-TV</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-Cont-Wavelet</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-Dynamic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PAPA-TV</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="lineEditNumDualIters">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="checkBoxScatterMap">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Use Scatter Map</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QLabel" name="labelScatterMap">
          <property name="text">
           <string>Scatter Map: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelScatterCoeff">
          <property name="text">
           <string>Scatter Coeff: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="lineEditScatterCoeff">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonSelectScatterMap">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="lineEditScatterMap">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="pushButtonSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRun">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Task"/>
    <addaction name="actionImport_Tasks"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Run</string>
    </property>
    <addaction name="actionRun_All_Tasks"/>
    <addaction name="actionRun_Current_Task"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_Task">
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport_Tasks">
   <property name="text">
    <string>Import Tasks...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionRun_All_Tasks">
   <property name="text">
    <string>Run All Tasks</string>
   </property>
  </action>
  <action name="actionRun_Current_Task">
   <property name="text">
    <string>Run Current Task</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
